### DISCUSSION TITLE
Any script to remove local toots mirrored from other instances?

### DISCUSSION THREAD

**Discussion by fivestone** [mastodon/mastodon#17880] [Q&A] [answered]
hi, I run an instance in a very small VPS. After joining a relay, the database as well as the cache storage increase rapidly. Now the Postgres is nearly 10GB and increasing. I think most of the toots mirrored in the local database are useless.

Is there any tootctl, or SQL script, to safely clean the local toots (e.g. older than 90days) which are mirrored from other instances, and not followed, retooted, replied by any local users?

Thanks a lot.

**Comment by ClearlyClaire** [comment, marked as answer]
There's `tootctl statuses remove` that does something like that. There are a few caveats, like it not deleting a toot if it has a reply, but it should help clean most of toots from relays.

**Reply by fivestone** [reply]
Thank you! That seems working well~
